common/src/main/java/com/microsoft/identity/common/internal/ui/browser/BrowserSelector.java

```diff
@@ -62,7 +62,7 @@ public class BrowserSelector {
     public static Browser select(final Context context, final List<BrowserDescriptor> browserSafeList) throws ClientException {
         final String methodTag = TAG + ":select";
         final List<Browser> allBrowsers = getAllBrowsers(context);
-        Logger.verbose(methodTag, "Select the browser to launch.");
+        Logger.info(methodTag, "Select the browser to launch from safeList size=" + browserSafeList.size());
```

Comment:
Log the size of allBrowsers as well. I'd prefer you log the actual package names but if that's a privacy concern I understand.

Reply:
Not a privacy concern - i just didn't want to flush the logs. 

We definitely can do that in the private test build though.